### DISCUSSION TITLE
How to display "no results" message when no result in autocomplete

### DISCUSSION THREAD

**Discussion by Organizer21** [Smile-SA/elasticsuite#2522] [Q&A] [answered]
It seem to me there's no quick easy way to do this, or am I just being retarded and blind. Would love to hear what if any solutions there are for solving this and in particular in regards to products, as that's the only results we display.

**Comment by Organizer21** [comment]
Nobody got a good approach for this or already had this issue and resolved it?

**Comment by amenk** [comment, marked as answer]
There seems no option for this - you need to overwrite the search_autocomplete template

**Reply by Organizer21** [reply]
Shocking that this is not a default option, I completely fail to see why, very disapointed and I'd love to understand the official reasoning (maybe just something I can't phantom).
